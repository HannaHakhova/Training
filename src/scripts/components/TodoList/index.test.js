import React from 'react';
import {shallow} from 'enzyme';
import TodoList from './index';

const container = shallow(<TodoList toggle={() => {}} todos ={[{
                    id: 1,
                    text: 'Do task1',
                    isDone: false
                },
                {
                    id: 2,
                    text: 'Do task2',
                    isDone: true
                },
                {
                    id: 3,
                    text: 'Do task3',
                    isDone: false
}]} />);

describe('TodoList component', () => {
    it('should match expected snapshot', () => {
        expect(container).toMatchSnapshot();
    });
});