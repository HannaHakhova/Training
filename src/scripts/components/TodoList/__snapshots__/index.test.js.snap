// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoList component should match expected snapshot 1`] = `
<ul>
  <Todo
    todo={
      Object {
        "id": 1,
        "isDone": false,
        "text": "Do task1",
      }
    }
    toggle={[Function]}
  />
  <Todo
    todo={
      Object {
        "id": 3,
        "isDone": false,
        "text": "Do task3",
      }
    }
    toggle={[Function]}
  />
  <Todo
    todo={
      Object {
        "id": 2,
        "isDone": true,
        "text": "Do task2",
      }
    }
    toggle={[Function]}
  />
</ul>
`;
